sql/binlog.cc

```diff
@@ -5320,7 +5321,9 @@ bool MYSQL_BIN_LOG::open_binlog(const char *log_name,
     std::string err_msg= "Either disk is full or file system is read only ";
     if (encrypt_binlog)
       err_msg+= "or encryption failed ";
-    err_msg+= "while opening the binlog. Aborting the server.";
+    err_msg+= "while opening the ";
```

Comment:
does this and the other err_msg+ need to be two spaces more to the right, and does the "if (encrypt_binlog)" need { } ? or is the if (encrypt_binlog) just applicable to err_msg+= "or encryption failed "; ? In which case this comment can be ignored :)

Reply:
This is correct.